javascript/javascript1/week4/homework_exercises/voice_assistant.js

```diff
@@ -0,0 +1,164 @@
+const userInfo = {
+    name: undefined,
```

Comment:
It is a good practice to initialize the properties of an object with default values that reflect their data type.

`const userInfo = {
    name: "", 
    todo: [],
};`